pkgs/applications/virtualization/kvmtool/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, fetchurl, lib }:
+
+stdenv.mkDerivation {
+  pname = "kvmtool";
+  version = "2021-12-14";
+
+  src = fetchgit {
+    url = "https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git";
+    rev = "cdd7d8cc0109bb8e2a0a04c5fe904b5ad4f07a80";
```

Comment:
Why use this commit and fetch a patch instead of building some newer commit?

Reply:
Right, there are newer commits! I was looking at https://github.com/kvmtool/kvmtool which hasn't synced up yet.